src/Microsoft.DotNet.Interactive.Tests/KeyValueStoreKernelTests.cs

```diff
@@ -201,8 +201,8 @@ public async Task when_from_url_is_used_with_content_then_the_response_is_not_st
 
             kernel
                 .FindKernel("value")
-                .As<DotNetKernel>()
-                .GetVariableNames()
+                .As<ISupportGetValue>()
+                .GetValueInfos()
                 .Should()
                 .NotContain("hi");
```

Comment:
If this still compiles then this is going to be an evergreen and incorrect test.

Reply:
?

Reply:
`GetValueInfos` returns a list of `ValueInfo`, which will never contain a string.

Reply:
done